### ISSUE TITLE
tsne color legend in divideBy should be able to have separate colors for different categories

### ISSUE THREAD

**Issue by karishma-gangwani** [stjude/proteinpaint#2124]
changing color for legend on one category changes the color on another. Can we separate this somehow?? so I could visualize different colors in different categories and make it more user-centric. 
for example this view in termdbtest, I changed the female and male colors to both shades of grey and both categories change color. 

![image](https://github.com/user-attachments/assets/a0b39a6b-d3a4-4669-80fd-a82977f2c6f6)

**Comment by xzhou82** [comment]
i don't think it reasonable to use different colors for male/female in the two side-by-side plots

**Comment by karishma-gangwani** [comment]
> i don't think it reasonable to use different colors for male/female in the two side-by-side plots

yes this is just an example. maybe not a good example, but other categories such as showing WT samples in grey vs mutated ones in some color. and having them side by side.
